packages/wxt/src/core/create-server.ts

```diff
@@ -107,10 +108,15 @@ async function createServerInternal(): Promise<WxtDevServer> {
 
       // Listen for file changes and reload different parts of the extension accordingly
       const reloadOnChange = createFileReloader(server);
+      const keyboardsShortCuts = createKeyBoardShortCuts(server);
       server.watcher.on('all', reloadOnChange);
+      keyboardsShortCuts.start();
     },
+
     async stop() {
       wasStopped = true;
+      const keyboardsShortCuts = createKeyBoardShortCuts(server);
```

Comment:
Shouldn't it be Shortcut and Keyboard? Because I don't think we're Short Cutting something lol

Reply:
If it is confusing I will change it.
What name do you suggest?

Reply:
```diff
- createKeyBoardShortCuts
+ createKeyboardShortcuts
```

"keyboard" and "shortcuts" are each one word, so you shouldn't capitalize the "b" and "c".